pkg/plugins/resources/k8s/store.go

```diff
@@ -162,9 +162,30 @@ func (s *KubernetesStore) List(ctx context.Context, rs core_model.ResourceList,
 	if err := s.Converter.ToCoreList(obj, rs, predicate); err != nil {
 		return errors.Wrap(err, "failed to convert k8s model into core counterpart")
 	}
+
+	total, err := s.countK8sResources(ctx, rs)
+	if err != nil {
+		return err
+	}
+	rs.GetPagination().SetTotal(uint32(total))
```

Comment:
@jakubdyszkiewicz here (ant the other similar places) I need a pointer to the Pagination struct so I can set the Total.